source/django-get-started/django-query-data.txt

```diff
@@ -0,0 +1,72 @@
+.. _django-get-started-query:
+
+==================
+Query MongoDB Data
+==================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: app, odm, code example
+
+You can import your models into the Python interactive shell
+to read data from the ``sample_mflix`` database.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Query the "users" collection for a specified email
+
+      Start a Python shell by running the following command:
+      
+      .. code-block:: bash
+
+         python3 manage.py shell
+      
+      Then, run the following code to query the 
+      ``sample_mflix.users`` collection for a movie viewer whose email is 
+      ``"jason_momoa@gameofthron.es"``:
+
+      .. code-block:: python
+
+         from sample_mflix.models import Movie, Viewer
+         
+         Viewer.objects.filter(email="jason_momoa@gameofthron.es").first()
+
+      This code returns the name of the matching user:
+
+      .. code-block:: bash
```

Comment:
Technically the formatter is "pycon" but since you don't format your console sessions normally, I'm not sure it matters.

Reply:
pycon is not an option for the .. code-block:: directive unfortunately